lib/src/commands/update.dart

```diff
@@ -13,7 +14,7 @@ class UpdateCommand extends Command<int> {
   /// {@macro update_command}
   UpdateCommand({
     required Logger logger,
-    PubUpdater? pubUpdater,
+    @visibleForTesting PubUpdater? pubUpdater,
```

Comment:
Is this for test only?

Reply:
Oops, I've realized my mistake. Sorry, I'll be making the required changes now.